src/main/mainWindow.js

```diff
@@ -70,6 +84,7 @@ function createMainWindow(config, options) {
   });
 
   const mainWindow = new BrowserWindow(windowOptions);
+  handleScreenResize(screen, mainWindow);
```

Comment:
How about using [`browser-window-created`](https://electron.atom.io/docs/api/app/#event-browser-window-created) of app module? All windows can get this feature.